index.html

```diff
@@ -1455,12 +1456,54 @@ <h3>
           <p>
             Given {{GamepadEffectParameters}} |params:GamepadEffectParameters|,
             a <dfn>valid dual-rumble effect</dfn> must have a valid
-            duration, a valid startDelay, and both the
+            {{GamepadEffectParameters/duration}}, a valid {{GamepadEffectParameters/startDelay}}, and both the
             {{GamepadEffectParameters/strongMagnitude}} and the
             {{GamepadEffectParameters/weakMagnitude}} must be in the range
             `[0,1]`.
           </p>
         </dd>
+        <dt>
+          <dfn data-for="GamepadHapticEffectType">"trigger-rumble" effect
+          type</dfn>
+        </dt>
+        <dd>
+          <p>
+            {{GamepadHapticEffectType/"trigger-rumble"}} describes a haptics
+            configuration with a vibration motor in each of the bottom front buttons
+            of a [=Standard Gamepad=] (buttons with [=canonical indexes=] 6 and 7) in addition to the two handle motors used for 
+            {{GamepadHapticEffectType/"dual-rumble"}}. Nowadays these buttons
+            most-commonly take the form of spring-loaded triggers. In this
+            configuration, either motor is capable of providing localized haptic
+            feedback on the button's surface.
+          </p>
+          <p>
+            A {{GamepadHapticEffectType/"trigger-rumble"}} effect is a
+            fixed-duration, constant-intensity vibration effect intended for an
+            actuator of this type. {{GamepadHapticEffectType/"trigger-rumble"}}
+            effects are defined by {{GamepadEffectParameters/startDelay}},
+            {{GamepadEffectParameters/duration}},
+            {{GamepadEffectParameters/strongMagnitude}},
+            {{GamepadEffectParameters/weakMagnitude}}, {{GamepadEffectParameters/leftTrigger}}, and {{GamepadEffectParameters/rightTrigger}}, none of which are
+            required because they default to 0.
+          </p>
+          <p>
+            {{GamepadEffectParameters/startDelay}}, {{GamepadEffectParameters/duration}},
+            {{GamepadEffectParameters/strongMagnitude}},
+            {{GamepadEffectParameters/weakMagnitude}} share the same definition with {{GamepadHapticEffectType/"dual-rumble"}}.
+            {{GamepadEffectParameters/leftTrigger}} and
+            {{GamepadEffectParameters/rightTrigger}}, respectively, set the
+            intensity levels for the left and right bottom front buttons
+            vibrations, normalized to the range `[0,1]`, defaulting to 0.
+          </p>
+          <p>
+            Given {{GamepadEffectParameters}} |params:GamepadEffectParameters|,
+            a <dfn>valid trigger-rumble effect</dfn> must have a valid
```

Comment:
Please update the definition of "valid effect" to also check for valid trigger-rumble effects. (Currently it only checks dual-rumble and will always consider trigger-rumble effects valid.)

Reply:
Thanks for pointing it out!

Reply:
@gabrielsanbrito, can you link to "valid effect", I think `[=valid effect|valid=]` should work...

Reply:
No. The dfn is fine as it is.

Reply:
Awesome! Did the same thing for the `dual-rumble` section too. Thanks!